src/_c_internal_utils.c

```diff
@@ -0,0 +1,61 @@
+#define PY_SSIZE_T_CLEAN
+#include <Python.h>
+#ifdef _WIN32
+#include <Objbase.h>
+#include <Shobjidl.h>
+#endif
+
+static PyObject* mpl_GetCurrentProcessExplicitAppUserModelID(PyObject* module)
+{
+#ifdef _WIN32
+  wchar_t* appid = NULL;
```

Comment:
We use 4-space indent in our C code, too.

Reply:
fixed